projects/core/src/lib/talks/talk-item/talk-item.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { TalkItemComponent } from "@jsmu/core";
+
+describe("TalkItemComponent", () => {
+	let component: TalkItemComponent;
+	let fixture: ComponentFixture<TalkItemComponent>;
+
+	beforeEach(async () => {
+		await TestBed.configureTestingModule({
+			declarations: [TalkItemComponent]
+		}).compileComponents();
+
+		fixture = TestBed.createComponent(TalkItemComponent);
+		component = fixture.componentInstance;
+		fixture.detectChanges();
+	});
+
+	it("should create", () => {
+		expect(component).toBeTruthy();
+	});
+
+	it("should compare length of talks array with null", function () {
+		component.ngOnInit();
+		expect(component.talks).not.toBeNull();
+	});
```

Comment:
1. the use of regular functions is not necessary, instead it is better to use an arrow
2.  ngOnInit should not be held responsible for "length of talks array"  and therefore should not be tested in this context
3. in any case  talks will not  null, if it not assign value it be undefined